src/game_api/script.cpp

```diff
@@ -168,6 +168,11 @@ ImVec2 screenify(ImVec2 pos)
     return screened;
 }
 
+bool exists(const std::string& filename) {
```

Comment:
Should probably use `std::filesystem::exists`

Reply:
oops! I forgot to remove that function! My apologies.